src/components/Accordion/Accordion.scss

```diff
@@ -92,3 +92,13 @@
 .c-accordion input[type='checkbox']:checked + label::after {
 	transform: rotate(135deg);
 }
+
+.accordion-aria-control-true {
```

Comment:
Please don't change RDS styles. 
Remove these classes.

Reply:
Checked, if remove the style, not works for single component. Original rds works, but in react doesn't work, need to manually close the div. one solution is to delete the item/add the item to have the expand/hidden animation, but seems doesn't meet the requirement. Any ideas?

Reply:
Please take a look on the current RDS component's code. You could find something there. :) 
https://github.com/ravendesignsystem/rds

Reply:
Thank you for your help, that problem has been fixed.